### PR TITLE
[ja] Reorg debug cluster/app pages

### LINKED ISSUE #33765: move /ja/docs/tasks/debug-application-cluster/get-shell-running-container/ to /ja/docs/tasks/debug/debug-application/get-shell-running-container/
**This is a Feature Request**

**What would you like to be added**

The English page has been moved. maybe need to update the page too.

**Why is this needed**

**Comments**

/language ja

### PR DESCRIPTION
**Pull request by kaitoii11**
Fix #33765 

Since the section has been reorganized in #32823 , I updated related documents as well.

### CODE REVIEW

File: content/ja/docs/tasks/debug/debug-application/debug-running-pod.md [hunk 1]
```diff
@@ -385,15 +390,260 @@ status:
...
+```
+
+例えば`kubectl exec`の`-i`と`-t`引数を使って、端末に接続されたシェルを実行することができます。
+
+```shell
+kubectl exec -it cassandra -- sh
+```
 
-## {{% heading "whatsnext" %}}
+詳しくは、[実行中のコンテナのシェルを取得する](/docs/tasks/debug/debug-application/get-shell-running-container/)を参照してください。
```
Comment (nasa9084) [inline content/ja/docs/tasks/debug/debug-application/debug-running-pod.md:431]:
```suggestion
詳しくは、[実行中のコンテナへのシェルを取得する](/ja/docs/tasks/debug/debug-application/get-shell-running-container/)を参照してください。
```

File: content/ja/docs/tasks/debug/debug-application/debug-running-pod.md [hunk 2]
```diff
@@ -385,15 +390,260 @@ status:
...
+
+{{< feature-state state="beta" for_k8s_version="v1.23" >}}
+
+{{< glossary_tooltip text="エフェメラルコンテナ" term_id="ephemeral-container" >}}は、コンテナがクラッシュしたり、コンテナイメージにデバッグユーティリティが含まれていないなどの理由で`kubectl exec`が不十分な場合に、対話的にトラブルシューティングを行うのに便利です([ディストロ・イメージ](
+https://github.com/GoogleContainerTools/distroless)の場合など)。
+
+### エフェメラルコンテナを使用したデバッグ例 {#ephemeral-container-example}
+
+実行中のPodにエフェメラルコンテナを追加するには、`kubectl debug`コマンドを使用することができます。
+まず、サンプル用のPodを作成します。
```
Comment (nasa9084) [inline content/ja/docs/tasks/debug/debug-application/debug-running-pod.md:443]:
```suggestion
まず、サンプル用のPodを作成します:
```

File: content/ja/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md [hunk 3]
```diff
@@ -38,4 +39,13 @@ Kubernetesは、Horizontal Pod Autoscalerなどのメカニズムを使用して
 CNCFプロジェクトの[Prometheus](https://prometheus.io)は、Kubernetes、ノード、Prometheus自身をネイティブに監視することができます。
 CNCFに属さない完全なメトリクスパイプラインのプロジェクトは、Kubernetesのドキュメントの範囲外です。
 
+## {{% heading "whatsnext" %}}
 
+以下のような追加のデバッグツールについて学びます:
+
+* [Logging](/docs/concepts/cluster-administration/logging/)
+* [Monitoring](/docs/tasks/debug-application-cluster/resource-usage-monitoring/)
```
Comment (nasa9084) [inline content/ja/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md:47]:
```suggestion
* [モニタリング](/ja/docs/tasks/debug-application-cluster/resource-usage-monitoring/)
```

File: content/ja/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md [hunk 4]
```diff
@@ -38,4 +39,13 @@ Kubernetesは、Horizontal Pod Autoscalerなどのメカニズムを使用して
...
+* [Getting into containers via `exec`](/docs/tasks/debug-application-cluster/get-shell-running-container/)
```
Comment (nasa9084) [inline content/ja/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md:48]:
```suggestion
* [`exec`でコンテナに入る](/ja/docs/tasks/debug-application-cluster/get-shell-running-container/)
```

File: content/ja/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md [hunk 5]
```diff
@@ -38,4 +39,13 @@ Kubernetesは、Horizontal Pod Autoscalerなどのメカニズムを使用して
...
+* [Connecting to containers via proxies](/docs/tasks/extend-kubernetes/http-proxy-access-api/)
+* [Connecting to containers via port forwarding](/docs/tasks/access-application-cluster/port-forward-access-application-cluster/)
+* [Inspect Kubernetes node with crictl](/docs/tasks/debug-application-cluster/crictl/)
```
Comment (nasa9084) [inline content/ja/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md:51]:
```suggestion
* [crictlでKubernetesのノードを検査する](/jadocs/tasks/debug-application-cluster/crictl/)
```

**Review by nasa9084 — COMMENTED** [review]
/assign

please rebase as well

**Comment by Sea-n** [comment]
Hey @kaitoii11,

This PR have `needs-rebase` label for 6 weeks. Would you like to solve it so that we can move forward?

**Comment by kaitoii11** [comment]
I apologize for the delay. For some reason, I thought I already resolved the rebase weeks ago.

File: content/ja/docs/tasks/debug/debug-application/debug-pods.md [hunk 6]
```diff
@@ -131,12 +132,12 @@ Podの`containerPort`がServiceの`targetPort`と一致することを確認し
...
+詳しくは[Serviceのデバッグ](/ja/docs/tasks/debug/debug-application/debug-service/)を参照してください。
 
 ## {{% heading "whatsnext" %}}
 
 上記のいずれの方法でも問題が解決しない場合は、以下の手順に従ってください。
-[Debugging Service document](/docs/tasks/debug-application-cluster/debug-service/)で、`Service` が実行されていること、`Endpoints`があること、`Pods`が実際にサービスを提供していること、DNS が機能していること、IPtablesルールがインストールされていること、kube-proxyが誤作動を起こしていないようなことを確認してください。
+[Serviceのデバッグに関するドキュメント](/ja/docs/tasks/debug/debug-application/debug-service/)で、`Service` が実行されていること、`Endpoints`があること、`Pods`が実際にサービスを提供していること、DNS が機能していること、IPtablesルールがインストールされていること、kube-proxyが誤作動を起こしていないようなことを確認してください。
 
-[トラブルシューティングドキュメント](/docs/tasks/debug-application-cluster/troubleshooting/)に詳細が記載されています。
+[トラブルシューティングドキュメント](/ja/docs/tasks/debug/)に詳細が記載されています。
```
Comment (nasa9084) [inline content/ja/docs/tasks/debug/debug-application/debug-pods.md:142]: again, this link returns 404 not found
https://kubernetes.io/ja/docs/tasks/debug/

Comment (kaitoii11) [reply content/ja/docs/tasks/debug/debug-application/debug-pods.md:142]: A new page linked to ja/docs/task/debug is created with this PR. content/ja/docs/tasks/debug/_index.md

I can see it has generated on the preview.
https://deploy-preview-34162--kubernetes-io-main-staging.netlify.app/ja/docs/tasks/debug/

Comment (nasa9084) [reply content/ja/docs/tasks/debug/debug-application/debug-pods.md:142]: ah sorry, so it's ok

File: content/ja/docs/tasks/debug/debug-application/debug-running-pod.md [hunk 7]
```diff
@@ -1,15 +1,20 @@
...
-しかし、アプリケーションに関するより多くの情報を取得する方法がいくつかあります。
+このページでは、ノード上で動作している(またはクラッシュしている)Podをデバッグする方法について説明します。
+
+
+## {{% heading "prerequisites" %}}
+
+
+* あなたの{{< glossary_tooltip text="Pod" term_id="pod" >}}は既にスケジュールされ、実行されているはずです。Podがまだ実行されていない場合は、[Troubleshoot Applications](/ja/docs/tasks/debug/debug-application/) から始めてください。
+
+* いくつかの高度なデバッグ手順では、Podがどのノードで動作しているかを知り、そのノードでコマンドを実行するためのシェルアクセス権を持っていることが必要です。`kubectl` を使用する標準的なデバッグ手順の実行には、そのようなアクセスは必要ではありません。
```
Comment (t-inu) [inline content/ja/docs/tasks/debug/debug-application/debug-running-pod.md:16]:
```suggestion
* いくつかの高度なデバッグ手順では、Podがどのノードで動作しているかを知り、そのノードでコマンドを実行するためのシェルアクセス権を持っていることが必要です。`kubectl`を使用する標準的なデバッグ手順の実行には、そのようなアクセスは必要ではありません。
```

File: content/ja/docs/tasks/debug/debug-cluster/audit.md [hunk 8]
```diff
@@ -1,6 +1,7 @@
 ---
 content_type: concept
 title: 監査
+weight: 30
```
Comment (t-inu) [inline content/ja/docs/tasks/debug/debug-cluster/audit.md:4]:
```suggestion
```
`audit.md` is not changed in en.

**Comment by t-inu** [comment]
Thank you for working on this complicated PR.
These are additional comments.

- I think `debug-application-cluster/debug-cluster.md` needs to be removed.

- `debug/debug-application/debug-service.md` was changed in en, so also needs to be changed in ja.

- `weight: 30` is required in `debug-application/debug-statefulset.md` and `debug/debug-cluster/crictl.md`.

- I seem there are more extensive changes in `debug/debug-cluster/_index.md`.

- `debug/debug-cluster/local-debugging.md` needs to be added "テレプレゼンスを使用した" as its title.

**Comment by t-inu** [comment]
There are some other content differences, but in this PR I'd like to address only the changes related to #32823.
To bring individual files up to date, I think it would be better to create new PRs after this PR is merged.

**Comment by t-inu** [comment]
I've confirmed about the fix for the [first review comments](https://github.com/kubernetes/website/pull/34162#pullrequestreview-1110077741). It's no problem.

Regarding 5 additional comments.
> - I think `debug-application-cluster/debug-cluster.md` needs to be removed.

I understood why this file doesn't display as "delete" at the left tree in the files page.
Because this file is "move" in Japanese,  on the other hand "delete" and "add" in English.
So, I withdraw this comment.

> - `debug/debug-application/debug-service.md` was changed in en, so also needs to be changed in ja.
> - `weight: 30` is required in `debug-application/debug-statefulset.md` and `debug/debug-cluster/crictl.md`.
> - `debug/debug-cluster/local-debugging.md` needs to be added "テレプレゼンスを使用した" as its title.

Could you please fix in this PR about above 3 comments?

> - I seem there are more extensive changes in `debug/debug-cluster/_index.md`.

There are large differences between this file and the original, we'd like to update the contents in a new PR.
However, in terms of changing the page structure, I think it would be better to add the following descriptions in this PR.
```
description: 一般的なクラスターの問題をデバッグします。
weight: 20
no_list: true
```

**Comment by kaitoii11** [comment]
I have added the changes you suggested.  And also added some changes in #34175 since it's affecting the changes I made.

File: content/ja/docs/tasks/debug/debug-application/debug-service.md [hunk 9]
```diff
@@ -591,11 +592,11 @@ UP BROADCAST RUNNING PROMISC MULTICAST  MTU:1460  Metric:1
...
-[Slack](/docs/tasks/debug-application-cluster/troubleshooting/#slack)、[Forum](https://discuss.kubernetes.io)または[GitHub](https://github.com/kubernetes/kubernetes)でお問い合わせください。
+[Slack](https://slack.k8s.io/)、[Forum](https://discuss.kubernetes.io)または[GitHub](https://github.com/kubernetes/kubernetes)でお問い合わせください。
 
 
 
 ## {{% heading "whatsnext" %}}
 
 
-詳細については、[トラブルシューティングドキュメント](/docs/tasks/debug-application-cluster/troubleshooting/)をご覧ください。
+詳細については、[トラブルシューティングドキュメント](/docs/tasks/debug/)をご覧ください。
```
Comment (t-inu) [inline content/ja/docs/tasks/debug/debug-application/debug-service.md:602]:
```suggestion
詳細については、[トラブルシューティングドキュメント](/ja/docs/tasks/debug/)をご覧ください。
```
This linked Japanese page will be created by this PR.

**Review by t-inu — COMMENTED** [review]
Great!
However, I have only one comment.

**Comment by t-inu** [comment]
Many thanks for the fix.
/lgtm

**Pull request merged**